dask/compatibility.py

```diff
@@ -9,14 +9,24 @@
 
 try:
     from dataclasses import is_dataclass, fields as dataclass_fields
-
 except ImportError:
-
+    # Python < 3.7
```

Comment:
Thanks for adding this comment, I can see it helping us in the future when we drop support for Python 3.7